### PR TITLE
Release 2.1.5

### PR DESCRIPTION
**Pull request by advornichenko-sift**
Release 2.1.5
- The Podspec iOS deployment target is set to 12.0

### CODE REVIEW

File: HelloSift/Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-github "SiftScience/sift-ios" ~> 2.1.2
\ No newline at end of file
+github "SiftScience/sift-ios" ~> 2.1.4
```
Comment (sbogolii-sift) [inline HelloSift/Cartfile:1]: it is correct 2.1.4 here ?

**Review by mpierotti-sift — APPROVED** [review]
LGTM

File: HelloSift/Cartfile (hunk 1 above)
Comment (mpierotti-sift) [reply HelloSift/Cartfile:1]: @sbogolii-sift Yes, this should point to the previous release.

Comment (advornichenko-sift) [reply HelloSift/Cartfile:1]: According to [runbook](https://sift.atlassian.net/wiki/spaces/RNDTEAM/pages/743309462/sift-ios+runbook#Pre-release-and-testing) Cartfile should point to the latest released in github (usually previous) version

**Review by sbogolii-sift — APPROVED** [review]
LGTM

**Pull request merged**